docs/modules/rotary.md

```diff
@@ -45,20 +44,18 @@ Initialize the nodemcu to talk to a rotary encoder switch.
 #### Returns
 Nothing. If the arguments are in error, or the operation cannot be completed, then an error is thrown.
```

Comment:
Should  rotary.setup function return object(switch)?

Reply:
Indeed it should. The perils of copying over documentation and not reading it carefully enough!